indy-utils/src/main/kotlin/com/luxoft/blockchainlab/hyperledger/indy/helpers/TailsHelper.kt

```diff
@@ -29,4 +32,25 @@ object TailsHelper {
 
         return BlobStorageHandler(reader, writer)
     }
+    class DefaultWriter (private val path: String) {
+        fun write(tailsResponse: TailsResponse) {
+            val dir = File(path)
+            if (!dir.exists())
+                dir.mkdirs()
+            tailsResponse.tails.forEach { name, content ->
+                val file = Paths.get(path, name).toFile()
+                if (file.exists())
+                    file.delete()
+                file.createNewFile()
+                file.writeText(content.toString())
+            }
+        }
+    }
+    class DefaultReader(private val path: String) {
+        fun read(tailsRequest: TailsRequest) : TailsResponse {
+            val file = Paths.get(path, tailsRequest.tailsHash).toFile()
```

Comment:
And whats up with tailsHash path injection fix?

Reply:
done